frontend/app/src/components/AccountManagement.vue

```diff
@@ -23,22 +23,23 @@
           </div>
         </div>
         <connection-loading
-          v-if="!connectionFailure"
+          v-if="!isConnectionFailed"
```

Comment:
can you change this to `hasConnectionFailed` instead

Reply:
updated